cpp/examples/deck.gl/flight-paths.cc

```diff
@@ -58,7 +58,7 @@ auto createLineLayer(const std::string &dataPath) -> std::shared_ptr<LineLayer::
   };
   lineLayerProps->getColor = [](const Row &row) -> mathgl::Vector4<float> {
     // NOTE: Decimal array data is being loaded as double
-    float z = static_cast<float>(row.getDoubleVector3("start").y);
+    float z = static_cast<float>(row.getDoubleVector3("start").z);
```

Comment:
Nit: if we had a `getFloatVector()` or `getVector<float>()` this code would look much cleaner.